storage-service/src/main/java/com/modsen/storage_service/config/SecurityConfig.java

```diff
@@ -39,7 +39,7 @@ public JwtAuthenticationConverter jwtAuthenticationConverter() {
                 .getOrDefault("roles", List.of())
                 .stream()
                 .map(SimpleGrantedAuthority::new)
-                .collect(Collectors.toList()));
+                .collect(toList()));
```

Comment:
По идее версия Java должна позволять сделать так:
```suggestion
                .toList();
```